index.html

```diff
@@ -3,9 +3,85 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Brewery Finder</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/bulma.sass">
+    <link rel="stylesheet" href= "https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.css">
+    <script src="https://use.fontawesome.com/releases/v5.14.0/js/all.js"></script>
+</head>
 </head>
 <body>
+    <header>
+        <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false">
+            <span aria-hidden="true"></span>
+            <span aria-hidden="true"></span>
+            <span aria-hidden="true"></span>
+          </a>
+        <section class="hero is-medium is-primary is-bold">
+            <div class="hero-body">
+              <div class="container">
+                <h1 class="title">
+                  A Hop, Skip, and a Jump
+                </h1>
+                <h2 class="subtitle">
+                  ...away from that best breweries in town.
+                </h2>
+              </div>
+            </div>
+          </section>
+    </header>
     
 </body>
+<div class="container">
+<div class="field has-addons">
+    <div class="control">
+      <input class="input" type="text" placeholder="Find a local brewery">
+    </div>
+    <div class="control">
+      <a class="button is-primary is-bold">
+        Search
+      </a>
+    </div>
+  </div>
+<div class="select">
+    <select>
+      <option>Select your brew</option>
+      <option>With options</option>
+    </select>
+  </div>
+  <div class="field">
+    <label class="label"></label>
+    <div class="control">
+      <input class="input" type="text" placeholder="Text input">
+    </div>
+    <p class="location">Call the location for details</p>
+  </div>
+  <div class="field">
+    <label class="label"></label>
+    <div class="control">
+      <input class="input" type="text" placeholder="Text input">
+    </div>
+    <p class="location">Call the location for details</p>
+  </div>
+  <div class="field">
+    <label class="label"></label>
+    <div class="control">
+      <input class="input" type="text" placeholder="Text input">
+    </div>
+    <p class="location">Call the location for details</p>
+  </div>
+  <div class="field">
+    <label class="label"></label>
+    <div class="control">
+      <input class="input" type="text" placeholder="Text input">
+    </div>
+    <p class="location">Call the location for details</p>
+  </div>
+  <div class="field">
+    <label class="label"></label>
+    <div class="control">
+      <input class="input" type="text" placeholder="Text input">
+    </div>
+    <p class="location">Call the location for details</p>
+  </div>
```

Comment:
What is this section for? Looks like its to contain info for the individual elements. I am going to dynamically render those sections in the index.js file. If that's the case, I can copy the structure of one of your div's here and use it in index.js

Reply:
Yes, that's what it was for.  If that works for you, that's ok.  If not, I can remove that section?